src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -177,7 +180,7 @@ class KeyframeRequester(
 
     override fun getNodeStats(): NodeStatsBlock {
         return super.getNodeStats().apply {
-            addString("wait_interval_ms", waitIntervalMs.toString()) // use string to prevent aggregation
+            addString("wait_interval_ms", waitInterval.toString()) // use string to prevent aggregation
```

Comment:
I think this is obsolete. We needed a string because the method used to take `Any` and aggregate numbers. But the new method takes a `String`.